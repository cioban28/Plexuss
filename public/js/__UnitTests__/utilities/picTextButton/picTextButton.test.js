import React from 'react';
// import PicTextButton from './../../../bundles/utilities/picTextButton/picTextButton';
import PicTextButton from './../../../bundles/utilities/picTextButton/picTextButton';
import renderer from 'react-test-renderer';

describe('PicTextButton', () => {

  it('pic button render test', () => {
      const component = renderer.create(
        <PicTextButton text="Facebook" imageClass="imgClass"></PicTextButton>
      );

      let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
  });


}); 