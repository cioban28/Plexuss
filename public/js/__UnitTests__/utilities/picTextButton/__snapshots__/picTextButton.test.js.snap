// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PicTextButton pic button render test 1`] = `
<div
  className="_picTextButton"
  onClick={undefined}
>
  <div
    className="btn-image"
  >
    <div
      className="imgClass"
    />
  </div>
  <div
    className="btn-text"
  >
    Facebook
  </div>
</div>
`;
